app.rb

```diff
@@ -62,6 +63,7 @@
 require './workers/import'
 require './workers/spider'
 require './workers/recon'
+require './workers/vulns'
```

Comment:
Also need to add this line to `workers.rb` so that sidekiq will also load the `Workers::Vulns` class.